main.go

```diff
@@ -91,55 +91,35 @@ func wrapMain() error {
 
 	fmt.Printf("Drone GAE Plugin built from %s\n", rev)
 
-	// https://godoc.org/github.com/drone/drone-plugin-go/plugin
-	workspace := plugin.Workspace{}
-	repo := plugin.Repo{}
-	build := plugin.Build{}
-	system := plugin.System{}
```

Comment:
none of these 3 were actually used, so they're not being copied into configFromStdin()